packages/utils/package.json

```diff
@@ -22,7 +22,12 @@
     "@babel/runtime": "^7.20.13",
     "@babel/runtime-corejs3": "^7.20.13",
     "@emotion/is-prop-valid": "1.2.0",
-    "@types/warning": "^3.0.0",
-    "warning": "4.0.3"
+    "tiny-warning": "1.0.3"
```

Comment:
`warning` logs console errors instead of warnings so I decided to use `tiny-warning` as it's what we are already using in `app-kit`.